### PR TITLE
Fix chunkify arrows on T2B1

### PR DESCRIPTION
**Pull request by grdddj**
Fixes https://github.com/trezor/trezor-firmware/issues/3271
- makes sure the arrow will always occupy the whole last/first chunk when paginating

### CODE REVIEW

File: core/embed/rust/src/ui/component/text/layout.rs [hunk 1]
```diff
@@ -264,7 +264,15 @@ impl TextLayout {
         {
             sink.prev_page_ellipsis(*cursor, self);
             // Move the cursor to the right, always the same distance
-            cursor.x += self.style.prev_page_ellipsis_width();
+            // Special case in chunkifying text - move the cursor so that we
+            // start with the second chunk.
+            if let Some(chunk_config) = self.style.chunks {
+                // Assuming mono-font, so all the letters have the same width
```
Comment (Hannsek) [inline core/embed/rust/src/ui/component/text/layout.rs:270]: That is not right, right? Regarding **w** and **m**.

**Comment by grdddj** [comment]
Looks like this:

image

Noticed just one place where the next page arrow is not completely visible - not sure why, would need to investigate this special case:

image

File: core/embed/rust/src/ui/component/text/layout.rs (hunk 1 above)
Comment (grdddj) [reply core/embed/rust/src/ui/component/text/layout.rs:270]: It does not matter here, we just want to offset the first chunk and the space (which combined should always be constant). 

(m, M, w and W are accounted for elsewhere, we are decreasing the space between chunks when we encounter them)

**Comment by Hannsek** [comment]
Cannot we put the "next-page" arrow to the left side of the last chunk and not the right? The same with the "previous-page" arrow. Put it on the right side instead of left.

File: core/embed/rust/src/ui/component/text/layout.rs (hunk 1 above)
Comment (Hannsek) [reply core/embed/rust/src/ui/component/text/layout.rs:270]: Wouldn't it change with [this](https://github.com/trezor/trezor-firmware/issues/3272)?

Comment (grdddj) [reply core/embed/rust/src/ui/component/text/layout.rs:270]: I am little confused with this second issue you mention here - what is the goal there?

**Comment by grdddj** [comment]
> Cannot we put the "next-page" arrow to the left side of the last chunk and not the right? The same with the "previous-page" arrow. Put it on the right side instead of left.
> 
> image

Right, corrected in the current state:

image

**Comment by Hannsek** [comment]
It does look ok. 👍

**Comment by Hannsek** [comment]
@matejcik review pls

**Review by matejcik — APPROVED** [review]
utACK

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: web 24.1.0 (50fe764c31b889a043eb335c4f7421bcb8d8541b)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:120.0) Gecko/20100101 Firefox/120.0
 - OS: MacIntel
 - Screen: 1915x1244
 - Device: Trezor T2B1 2.6.4 regular (revision 1f693d3561c375661563970e066f678590abc19c)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**